src/main/java/com/solvd/laba/carina/products/utils/TestUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.solvd.laba.carina.products.utils;
+
+import com.solvd.laba.carina.products.gui.pages.HomePage;
+import com.zebrunner.carina.core.IAbstractTest;
+
+public class TestUtil implements IAbstractTest {
```

Comment:
this can also be moved to the test package in an abstract parent class with @BeforeMethod since each test implements this operation

Reply:
Thanks for the suggestions. I have created an abstract class with BeforeMethod.